.cirrus.yml

```diff
@@ -10,6 +10,8 @@ task:
     - pkg install -y curl
     - curl https://sh.rustup.rs -sSf --output rustup.sh
     - sh rustup.sh -y
```

Comment:
By that point it's too late, because line 12 already installs the default profile.  Do `sh rustup.sh -y --profile=minimal` instead.

Reply:
Ah, didn't know that. I've force pushed since the change is so small. Let's see if this does have an impact.